README.md

```diff
@@ -1,3 +1,28 @@
+MSG TO ALLA (developer)
+
+// github:
```

Comment:
ok I get! (alla said) 

Can you please take this out before we merge it!

I will create a better read me, during my FIRST commit (all)